### DISCUSSION TITLE
Where is a definitive guide?

### DISCUSSION THREAD

**Discussion by jborys** [getbem/getbem.github.io#228] [Q&A]
Where can I find a very clear and concise guide with examples on how to write a block, and element, and a modifier. And how each is identified and used?

**Comment by lost-diver** [comment]
Would like this too.
The BEM website gives a good theoretical understanding of the system

**Comment by dotnetCarpenter** [comment]
I think that https://getbem.com/naming/ gives the most clear and concise explanation but there is still unanswered questions like mixes (https://en.bem.info/methodology/css/#mixes).
